client/src/styles/style.less

```diff
@@ -6,7 +6,6 @@
 @import "_forms";
 @import "_buttons";
 @import "_modal";
-@import "_status-bar";
```

Comment:
We define color variables for tabs (`_tabs.less`) and overlays (`_overlays.less`) also separately. We should be consistent, having these separate files, or keeping all in the individual components (like in this PR for the status bar).

Having the variables under `styles/` has the advantage we have them close to other variables (font, colors).

Reply:
Personally I'm always searching for variables locally (to the file). This works fine for me, especially given that _local_ variables are already namespaced accordingly, too.

Re-using a variable (i.e. `tabbed-xxx`) outside the component shall be an anti-pattern to avoid. Especially because of that I'd go for local vars as a pattern and only define vars under `styles` if they are actually global (i.e. ought to be re-used).

Happy to make the adjustments (tabs, overlays) if you think such direction makes sense. 

Would also be interested in your feedback here @smbea.

Reply:
Given I would have a look at a certain component the first time, the things you've mentioned make absolute sense to me --> If I see a variable, I should quickly find out where it is defined, preferably just locally next to the component itself. So I'd agree to have it the way you proposed. 

Generally, I'm fine with everything we came up with, as long we are consistent 🙂